web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -601,7 +600,7 @@ const TeamReviews = ({ onBack, periodId }) => {
     return null; // no validtation errors
   };
 
-  const launchReviewPeriod = async () => {
+  const updateReviewPeriodStatus = async (reviewStatus) => {
```

Comment:
I renamed this function and added a parameter so I can update the review status differently based on that value.